docker-compose.yaml

```diff
@@ -0,0 +1,39 @@
+services:
+  kt-app:
+     image: jakefrosty/korotagger:1.3
+     cpus: 0.5
+     mem_limit: 100m
+     container_name: korotagger-app
+     environment:
+       yt_api_key: your-yt-api-key
```

Comment:
could you list the environment variable a user would be expected to set in the readme, this would be useful for people running outside docker as well

Reply:
I will make a new make an updated README.md sure!
I'll include syntax for directly running and via docker aswell